app/app.R

```diff
@@ -478,6 +478,10 @@ ui <- (
                            "max-options" = 8), 
             multiple = TRUE)
           
+        ),
+        conditionalPanel(
+          condition = "input.y_type == 'Income Components'",
+          p(em("*Income components represent the average amount per population unit in each quantile/band, and are rounded to the nearest $1000. Thus, if the amount recieved on average by population units in each ventile/income band is too small, the income component will not be displayed. It may be useful to select a more targeted population subgroup to explore the contribution of this component to incomes."))
```

Comment:
typo: recieved -> received

Reply:
Thanks Luke, have fixed this. I have also updated the hyperlinks - isn't a perfect fix as it doesn't work in the RStudio environment but does work in web browsers which are our main target at the moment.